packages/flutter/lib/src/material/slider.dart

```diff
@@ -1997,7 +2051,7 @@ class _SliderDefaultsM2 extends SliderThemeData {
 
 class _SliderDefaultsM3 extends SliderThemeData {
   _SliderDefaultsM3(this.context)
-    : super(trackHeight: 4.0);
+    : super(trackHeight: 16.0);
```

Comment:
Why the change here, do we expect it to be breaking? Is this value represented by a token?

Reply:
Should this number be kept in a constant?

Reply:
> Why the change here, do we expect it to be breaking? Is this value represented by a token?

This is updated by the  track height token which has a new value. 

```dart
class _${blockName}DefaultsM3 extends SliderThemeData {
  _${blockName}DefaultsM3(this.context)
    : super(trackHeight: ${getToken('$tokenGroup.active.track.height')});

```

Reply:
Perfect, thanks!